crypto/ec/ec_key.c

```diff
@@ -119,9 +119,11 @@ EC_KEY *EC_KEY_copy(EC_KEY *dest, const EC_KEY *src)
                     return NULL;
             }
             if (!BN_copy(dest->priv_key, src->priv_key))
+                BN_free(dest->priv_key);
```

Comment:
I think the reason our tools report this issue and change is because it did not know dest would always be freed. But if this can be ensured, then this would not be an issue.